File: Tugas B/cpp/hardware.cpp
#include "product.cpp"

class hardware : public product{
	private:
		string brand;
        string model;
	
	public:
		//Construct
		hardware(){}
		
		hardware(string brand, string model){
			this->brand = brand;
			this->model = model;
		}
		
		//Set dan Get
		void setbrand(string brand)
        {
            this->brand = brand;
        }
		string getbrand(){
			return this->brand;
		}
		
		void setmodel(string model){
			this->model = model;
		}
		string getmodel(){
			return this->model;
		}
		
		//Output
		void output(){
			cout << "Brand	 	: " << this->getbrand() << endl;
			cout << "Model	 	: " << this->getmodel() << endl;
			cout << "Price	 	: " << this->getprice() << endl;
			cout << "ID 		: " << this->getid() << endl;
			cout << "Frekuensi       : " << this->getfreq() << endl;
			cout << "Size		: " << this->getsize() << endl;
			cout << "Cuda 		: " << this->getcuda() << endl;
		}
		
		~hardware(){
		}
};

File: Tugas B/php/hardware.php
<?php

include "product.php";

class hardware extends product
{
	
	private $brand = "";
	private $model = "";
	
	
	//Construct
	public function __construct($brand = "", $model = ""){
		$this->brand = $brand;
		$this->model = $model;
	}
	
	//Set dan get
	public function setbrand($brand){
		$this->brand = $brand;
	}
	public function getbrand(){
		return $this->brand;
	}
	
	public function setmodel($model){
		$this->model = $model;
	}
	public function getmodel(){
		return $this->model;
	}
	
	//output print
	public function output(){
		echo "Brand 	: ". $this->getbrand(). "<br/>";
        echo "Model 	: ". $this->getmodel(). "<br/>";
		echo "Price 	: ". $this->getprice(). "<br/>";
        echo "ID        : ". $this->getid(). "<br/>";
		echo "Frekuensi : ". $this->getfreq(). "<br/>";
        echo "Size 		: ". $this->getsize(). "<br/>";
        echo "Cuda 		: ". $this->getcuda(). "<br/>";
	}
	
}
?>